### DISCUSSION TITLE
Not sure how to use the new settings for hardware

### DISCUSSION THREAD

**Discussion by olskar** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#508] [Q&A]
Hello!

I recently flashed the new firmware, as I understand it one should not have to define anything in the code anymore but everything can be setup in the UI.

My model is 54112, and worked fine earlier with NO54123 defined.

However it does not work with the new firmware even though I choose NO54123 in the hardware settings.

Any ideas?

**Comment by visualapproach** [comment]
Did you select the correct PCB/pin out also, and then save+restart?

**Reply by olskar** [reply]
![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/18314560/54f1b493-eea4-488e-b0b7-d62379559134)

This is the settings I have. I never did anything to the pin out in the previous firmware, is it possible I need to change them from default now?

And this is the PCB I have 
![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/18314560/2517a852-f21c-4527-9e98-eda36cc0aecd)

**Reply by visualapproach** [reply]
What if you choose custom and put 3-2
6-7
?

**Reply by visualapproach** [reply]
You can try all 4 combinations. I don't think the original was wrong. Will check later.
What is not working by the way?

**Reply by nono721987** [reply]
i have the same pump and tried everything but my pcb ist v2b @olskar  can you send me a picture of the wiring? what version were you using when it worked?
